src/components/mobile/SettingsPage.js

```diff
@@ -63,6 +64,12 @@ const SettingsPage = () => (
         </AccordionPanel>
       </AccordionItem>
     </SettingsAccordion>
+
+    <Slider
+      style={{ marginLeft: 40, marginRight: 40 }}
```

Comment:
Consider making this one rule `{{ margin: '0 40px' }}`

Reply:
So, this is still in-line style but style={{ margin: '0 40px' }} is what you're saying, right?

Reply:
Okay I didn't look at the file title lmao my bad. You mean to the slider itself lol